<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResourceMonitorClass</class>
 <widget class="QDialog" name="ResourceMonitorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ResourceMonitor</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="simple">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label4">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>252</y>
       <width>291</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>250</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label1">
        <property name="text">
         <string>Memory:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label2">
        <property name="text">
         <string>3.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label3">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="detailed">
    <attribute name="title">
     <string>é¡µ</string>
    </attribute>
    <widget class="QCustomPlot" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>581</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_down">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>300</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/new/prefix1/C:/Users/54628/Downloads/down.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_on">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>300</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/new/prefix1/C:/Users/54628/Downloads/on.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelDown">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>310</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelOn">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>310</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>302</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="ResourceMonitor.qrc">:/new/prefix1/Resources/down.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>302</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="ResourceMonitor.qrc">:/new/prefix1/Resources/on.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>302</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="ResourceMonitor.qrc">:/new/prefix1/Resources/cpu.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelCpu">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>310</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>302</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="ResourceMonitor.qrc">:/new/prefix1/Resources/memory.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelMemory">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>310</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="setting">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>310</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="submit">
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="quit">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>setting</string>
     </property>
    </widget>
    <widget class="QLabel" name="label6">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>setting2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="dataSave">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Add data to file</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="highDensity">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Gride line high density display</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>84</x>
       <y>180</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Cpu line</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cpuLine">
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="maxVisibleItems">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>312</x>
       <y>180</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label8">
        <property name="text">
         <string>Memory line</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="memoryLine"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>240</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label9">
        <property name="text">
         <string>Down speed line</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="downSpeedLine"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>304</x>
       <y>240</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label10">
        <property name="text">
         <string>On speed line</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="onSpeedLine"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ResourceMonitor.qrc"/>
 </resources>
 <connections/>
</ui>
